community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/DeleteAcceptanceTest.scala

```diff
@@ -71,4 +71,27 @@ class DeleteAcceptanceTest extends ExecutionEngineFunSuite with QueryStatisticsT
 
     assertStats(result, nodesDeleted = 4, relationshipsDeleted = 3)
   }
+
+  test("expand, delete and count") {
```

Comment:
These tests are dependant on the direction of the match being `BOTH`, right? Perhaps we should include this in the test name.